src/LuaTransformer.ts

```diff
@@ -3033,7 +3113,16 @@ export class LuaTransformer {
             || this.currentNamespace
             || (tsOriginal && tsHelper.findFirstNodeAbove(tsOriginal, ts.isFunctionLike))
         ) {
-            statements.push(tstl.createVariableDeclarationStatement(lhs, undefined, parent));
+            if (!isArray(lhs)) {
```

Comment:
Is this `isArray` correct? I thought it was `Array.isArray`, or where is this defined?

Reply:
its defined in node's util lib. But since we use Array.isArray in other places I will change it to Array.isArray swell.